### ISSUE TITLE
Duplicate effects

### ISSUE THREAD

**Issue by frenck** [Aircoookie/WLED#294]
The `Chase` effect is on the list twice. For a normal user that is indistinguishable.

```json
{
    "effects": [
        "Solid",
        "Blink",
        "Breathe",
        "Wipe",
        "Wipe Random",
        "Random Colors",
        "Sweep",
        "Dynamic",
        "Colorloop",
        "Rainbow",
        "Scan",
        "Dual Scan",
        "Fade",
        "Chase",
        "Chase Rainbow",
        "Running",
        "Saw",
        "Twinkle",
        "Dissolve",
        "Dissolve Rnd",
        "Sparkle",
        "Dark Sparkle",
        "Sparkle+",
        "Strobe",
        "Strobe Rainbow",
        "Mega Strobe",
        "Blink Rainbow",
        "Android",
        "Chase",
        "Chase Random",
        "Chase Rainbow",
        "Chase Flash",
        "Chase Flash Rnd",
        "Rainbow Runner",
        "Colorful",
        "Traffic Light",
        "Sweep Random",
        "Running 2",
        "Red & Blue",
        "Stream",
        "Scanner",
        "Lighthouse",
        "Fireworks",
        "Rain",
        "Merry Christmas",
        "Fire Flicker",
        "Gradient",
        "Loading",
        "In Out",
        "In In",
        "Out Out",
        "Out In",
        "Circus",
        "Halloween",
        "Tri Chase",
        "Tri Wipe",
        "Tri Fade",
        "Lightning",
        "ICU",
        "Multi Comet",
        "Dual Scanner",
        "Stream 2",
        "Oscillate",
        "Pride 2015",
        "Juggle",
        "Palette",
        "Fire 2012",
        "Colorwaves",
        "BPM",
        "Fill Noise",
        "Noise 1",
        "Noise 2",
        "Noise 3",
        "Noise 4",
        "Colortwinkle",
        "Lake",
        "Meteor",
        "Smooth Meteor",
        "Railway",
        "Ripple",
        "Twinklefox"
    ]
}
```

**Comment by Aircoookie** [comment]
Good catch! I will rename FX 13 to Theater Chase, I believe!

**Comment by frenck** [comment]
`Chase Rainbow`, another one!

**Comment by frenck** [comment]
Fixed in ddaaae46a6922cc79056756017be1eb1f431417b
